exporters/etw/include/opentelemetry/exporters/etw/etw_logger.h

```diff
@@ -359,19 +365,30 @@ class LoggerProvider : public opentelemetry::logs::LoggerProvider
 
   nostd::shared_ptr<opentelemetry::logs::Logger> GetLogger(
       opentelemetry::nostd::string_view logger_name,
-      opentelemetry::nostd::string_view library_name,
-      opentelemetry::nostd::string_view version    = "",
-      opentelemetry::nostd::string_view schema_url = "",
+      opentelemetry::nostd::string_view library_name = "",
+      opentelemetry::nostd::string_view version      = "",
+      opentelemetry::nostd::string_view schema_url   = "",
       const opentelemetry::common::KeyValueIterable &attributes =
           opentelemetry::common::NoopKeyValueIterable()) override
   {
-    UNREFERENCED_PARAMETER(library_name);
     UNREFERENCED_PARAMETER(version);
     UNREFERENCED_PARAMETER(schema_url);
     UNREFERENCED_PARAMETER(attributes);
+
+    std::string event_name{ETW_VALUE_LOG};
+    if (config_.enableTableNameMappings)
+    {
+      auto it =
+          config_.tableNameMappings.find(std::string(library_name.data(), library_name.size()));
```

Comment:
std::string() will create a copy, can we use nostd::string_view instead? Also, unordered_map instead of map?

Reply:
Resolve this for now as this is not in the logging hot path.